vast/losses/losses.py

```diff
@@ -137,30 +137,27 @@ def __call__(self, logit_values, target, sample_weights=None):
 
 
 class objectoSphere_loss:
-    def __init__(self, batch_size, knownsMinimumMag=50.0):
+    def __init__(self, batch_size = None, knownsMinimumMag=50.0): # batchsize ignored for backward compatibility
```

Comment:
I think we do not need backward compatibility here since this feature isn't used by many so just have to change my own code.

Reply:
I have removed the `batch_size` here and in the `MNIST_Experiments` repo.